File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let countDict = {}
        for (let num of nums){
            if (num in countDict){
                return true
            } else {
                countDict[num] = 1
            }
        }
        return false
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numSet = set(nums)
        global_max = 0

        for num in nums:
            if num-1 not in numSet:
                curr = num
                length = 1
                while curr + 1 in numSet:
                    length += 1
                    curr += 1
                if length > global_max:
                    global_max = length
        
        return global_max
            

